webapp/services/import_services/ocpi/ocpi_validators.py

```diff
@@ -125,7 +126,7 @@ class GeoLocationInput(ValidataclassMixin):
 @validataclass
 class DisplayTextInput(ValidataclassMixin):
     language: str = StringValidator(min_length=2, max_length=2)
-    text: str = StringValidator(max_length=512)
+    text: str = ReplacingStringValidator(mapping={'\r': '', '\n': ' ', '\xa0': ' '}, max_length=512)
```

Comment:
the `\xa0` looks like a non-breaking-space, is it breaking anything (no pun intended)? otherwise I'd leave it as it is.

Reply:
I just copied that mapping from some other place as it was suggested... not sure if the current data even contain those, actually.
I don't know all the places where data of this `DisplayTextInput` type are used, and if all of them can handle non-breaking spaces, so I just stuck to the recommended default.